File: lua/refactoring/tests/refactor/106/go/simple-function/extract.expected.go
package main

import "fmt"

func foo_bar(a int, test INSERT_PARAM_TYPE, test_other INSERT_PARAM_TYPE) {
	for idx := test - 1; idx < test_other; idx++ {
		fmt.Println(idx, a)
	}
}

func simple_function(a int) {
	var test int = 1
	test_other := 1
	foo_bar(a, test, test_other)

}

File: lua/refactoring/tests/refactor/106/cpp/simple-function/extract.expected.cpp
#include <iostream>

void foo_bar(int a, int test, int test_other) {
  for (int idx = test - 1; idx < test_other; idx++) {
    std::cout << idx << " " << a << std::endl;
  }
}


void simple_function(int a) {
  int test = 1, test_other = 1;

  foo_bar(a, test, test_other);
}
